Rmd/Sample_QC.Rmd

```diff
@@ -264,37 +271,21 @@ if (file.exists(remove_file)) {
 
 # Sample Filtering Code
 
-```{r read-count-filter-not-used, echo = F, eval = F}
-############################################################################
-# Filter by median read counts per gene per group
-############################################################################
-# Nominally, this is done for the groups under consideration instead.
-# Done for each contrast within the RMarkdown script,
-# so no need to run it here.
-# 
-# group <- unique(d$group)
-# group <- group[order(group)]
-# 
-# #Way too many groups, lets use some parallel computing on this
-# #okay this is much faster
-# 
-# cores = detectCores()
-# cl <- makeCluster(cores[1]-1) #not to overload your computer
-# registerDoParallel(cl)
-# medcounts <- foreach(k = 1:length(group), .combine = cbind) %dopar%
-#   apply(sampleData[,d$group == group[k]], 1, median)
-# 
-# stopCluster(cl)
-# 
-# gene_flag <- apply(medcounts, 1, max)
-# quantile(gene_flag, probs = seq(0.1, 0.5, by = 0.1))
-# 
-# gene_flag <- gene_flag > 5
-# 
-# nrow(sampleData)
-# sampleData <- sampleData[gene_flag,]
-# p <- row.names(sampleData)
-# nrow(sampleData)
+## Samples Removed Manually
+
+If samples were removed due to the results of cytotoxicity assays or other experimentally-determined reasons, they are listed here.
+
+In this experiment, `r nrow(cytotoxic)` samples were removed manually, from the following groups:
+
+`r if (!is.null(remove)) { print(remove) }`
+
+```{r}
+
+cytotoxic %>%
+  kable(caption = "Samples removed manually before this report was generated.") %>%
+  kable_styling(bootstrap_options = "striped", full_width = F, position = "left") %>%
+  scroll_box(width = "100%", height = "480px")
+
```

Comment:
I'm confused as to why this is showing up here when these changes were already merged into main from the other branch. Hopefully you don't have any issues merging.

Reply:
Hmm. Good point. Says no conflicts... guess we will see

Probably because I was pulling one branch into another... but yeah since the other one was merged I would think this shouldn't be an addition